content/release-10-17-0/devices-10-17-0.md

```diff
@@ -149,6 +170,21 @@ mandatory <b>Software type</b> field is filled.</td>
 <td>10.16.185.0</td>
 </tr>
 
+<tr>
+<td>LWM2M</td>
+<td>A new LWM2M device configuration option has been added to control if the persisted values in the <b>Objects</b> tab get removed in case a read, write or observe observation fails. The option is configurable in the new single device registration in the advanced settings or by having the "c8y_GenericUI_retainOldValuesIfError" fragment in the LWM2M device managed object. By default the persisted values are kept even when the device operation fails. When this is disabled the read, write or observe operations failing with 4.01 (Unauthorized),4.03 (Forbidden), 4.04 (Not found), 5.00 (Internal Server Error), or 5.01 (not implemented) will lead to removal of the corresponding LWM2M path from the <b>Objects</b> tab.
```

Comment:
```suggestion
<td>A new LWM2M device configuration option has been added to control if the persisted values in the <b>Objects</b> tab get removed in case a read, write or observe observation fails. The option is configurable in the new single device registration in the advanced settings or by having the "c8y_GenericUI_retainOldValuesIfError" fragment in the LWM2M device managed object. By default the persisted values are kept even when the device operation fails. When this is disabled the read, write or observe operations failing with 401 (Unauthorized),403 (Forbidden), 404 (Not found), 500 (Internal Server Error), or 501 (not implemented) will lead to removal of the corresponding LWM2M path from the <b>Objects</b> tab.
```